.github/workflows/test.yml

```
@@ -113,15 +99,33 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - uses: actions/download-artifact@v2
-        with:
-          name: parallel-lint-phar
-
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
-          coverage: false
+          coverage: none
+          tools: cs2pr
+
+      - name: Install Composer dependencies
+        uses: ramsey/composer-install@v1
+
+      - name: 'Run unit tests PHP 5.4, 5.5'
+        if: ${{ matrix.php < 5.6 }}
+        run: composer testphp5
+
+      - name: 'Run unit tests PHP >= 5.6'
+        if: ${{ matrix.php >= 5.6 }}
+        run: composer test
+
+      - name: 'Integration test 1 - linting own code, no colors'
+        run: ./parallel-lint --exclude vendor --exclude tests/examples --no-colors .
+
+      - name: 'Integration test 2 - linting own code'
+        run: ./parallel-lint --exclude vendor --exclude tests/examples . --checkstyle | cs2pr
```

The violations will not be reported if the first integration check fails, but I guess you'd want this to show in the report too right?

Maybe, we can always run integration test 2 if integration test 1 fails or passes, but not if it's skipped (failure early on). I'm guessing this might work.

```suggestion
      - name: 'Integration test 1 - linting own code, no colors'
        id: integration-test-1
        run: ./parallel-lint --exclude vendor --exclude tests/examples --no-colors .

      - name: 'Integration test 2 - linting own code'
        if: ${{ steps.integration-test-1.conclusion == 'failure' || steps.integration-test-1.conclusion == 'success' }}
        run: ./parallel-lint --exclude vendor --exclude tests/examples . --checkstyle | cs2pr
```